Assets/MRTK/NOTICE.md

```diff
@@ -0,0 +1,97 @@
+NOTICES AND INFORMATION
```

Comment:
Instead of having multiple copies of each of the .md files (which could get unwieldy when we need to update the contents in all places), can we just have multiple copies of the .meta files that get copied in during packaging? And copy the existing .md files from the root for all packages

Reply:
In a different project, I had a UnityMetas folder in the CI/packaging scripting folder and copied then in when relevant. We could have something like NOTICE.md.meta.examples and `Copy-Item -Path "$projectRoot/packaging/UnityMetas/LICENSE.md.meta.$entry" -Destination "$packagePath/LICENSE.md.meta"` or wherever

Reply:
i'll do something like that

Reply:
done